symfony/messenger/4.3/config/packages/messenger.yaml

```diff
@@ -0,0 +1,14 @@
+framework:
+    messenger:
+        # Uncomment the following line and choose a transport to collect messages that fail so you can retry them later
+        # failure_transport: failed
+
+        transports:
+            # https://symfony.com/doc/current/messenger.html#transports
+            # async: '%env(MESSENGER_TRANSPORT_DSN)%'
+            # failed: '%env(MESSENGER__FAILED_TRANSPORT_DSN)%'
```

Comment:
Someone might disagree, but I don’t think we need a separate environment variable for this. Rather, I think we should literally use doctrine:///default?queue_name=default

We’ll document that you could also reuse the main env var here and pass other options, like if you wanted to reuse your Amqp config, but use a different exchange for failed messages.

Reply:
I agree